plugins/org.locationtech.udig.catalog.ui/src/org/locationtech/udig/catalog/service/database/DatabaseServiceDialect.java

```diff
@@ -1,13 +1,13 @@
 /*
- *    uDig - User Friendly Desktop Internet GIS client
- *    http://udig.refractions.net
- *    (C) 2012, Refractions Research Inc.
- *
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
- * License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
- */
+*    uDig - User Friendly Desktop Internet GIS client
+*    http://udig.refractions.net
+*    (C) 2012, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```

Comment:
here the like define in codetemplates.xml we should keep the space at the beginning